configure.ac

```diff
@@ -67,24 +67,24 @@ AC_ARG_VAR([OMR_PRODUCT_DESCRIPTION],
   [A description of the product which is integrating OMR.] (Default: AC_PACKAGE_STRING AC_PACKAGE_URL))
 
 AC_ARG_VAR([OMR_ZOS_COMPILE_ARCHITECTURE],
-  [The z/OS compile machine architecture. (Default: 7)])
+  [The z/OS compile machine architecture. (Default: 10)])
 AC_ARG_VAR([OMR_ZOS_COMPILE_TARGET],
-  [The z/OS compile target operating system. (Default: zOSV1R13)])
+  [The z/OS compile target operating system. (Default: ZOSV2R3)])
 AC_ARG_VAR([OMR_ZOS_COMPILE_TUNE],
   [The z/OS compile machine architecture tuning. (Default: 10)])
```

Comment:
Should we update this to 12 as well?

Reply:
sorry I missed this on my last update, updated now.